docs/sources/datasources/influxdb/query-editor/index.md

```diff
@@ -2,14 +2,14 @@
 aliases:
   - ../../data-sources/influxdb/query-editor/
   - influxdb-flux/
-description: Guide for Flux in Grafana
+description: This topic describes the InfluxDB query editor, modes and querying the InfluxDB data source.
 labels:
   products:
     - cloud
     - enterprise
     - oss
-title: Query Editor
-weight: 200
+title: InfluxDB query Editor
```

Comment:
Shorter menu item for the table of contents where the hierarchy implies the "InfluxDB" part

```suggestion
title: InfluxDB query Editor
menuTitle: Query editor
```